addons/website_event_track/views/event_track_templates_page.xml

```diff
@@ -202,7 +214,7 @@
     <div class="d-flex align-items-center">
         <small class="text-muted" t-esc="track_other.partner_name"/>
         <div class="d-inline-block ml-auto o_wesession_track_aside_info text-truncate">
-            <small t-if="not track_other.website_published and user_event_manager"
+            <small t-if="not track_other.website_published and env.user.has_group('event.group_event_manager')"
```

Comment:
Why this change ? Is the "user_event_manager" variable invalid ?

Reply:
Reading again, seems like a fix we should backport to stable, isn't it ? Variable user_event_manager does not seem to exist anymore.